sourcemap_test.go

```diff
@@ -58,3 +60,38 @@ func TestWriteTo(t *testing.T) {
 		t.Error(b.String())
 	}
 }
+
+func TestWriteToShouldIncludeEmptyNames(t *testing.T) {
+	m, err := ReadFrom(strings.NewReader(testFileWithoutNames))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if m.Names != nil {
+		t.Error("Names expected to be nil")
+	}
+	b := bytes.NewBuffer(nil)
+	if err := m.WriteTo(b); err != nil {
+		t.Fatal(err)
+	}
+	if !strings.Contains(b.String(), `"names":[]`) {
+		t.Error(b.String())
```

Comment:
Please add at least a small error description, so the error message is not only the json.

Reply:
👍 Can do!